app/controllers/user/properties_controller.rb

```diff
@@ -19,10 +19,8 @@ def create
   end
 
   def index
-    if current_user.properties
-      @properties = current_user.properties
-      @amenities = Amenity.all
-    end
+    @properties = current_user.properties
```

Comment:
nice! I see that you took out the "if" statement since only logged_in users will ever even get to this page. Nice work!